apps/wallet/src/ui/app/pages/home/nft-transfer/index.tsx

```diff
@@ -1,124 +1,68 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { getObjectId, hasPublicTransfer } from '@mysten/sui.js';
-import cl from 'classnames';
-import { Formik } from 'formik';
-import { useCallback, useMemo, useState } from 'react';
+import {
+    getObjectId,
+    hasPublicTransfer,
+    is,
+    SuiObject,
+    getObjectOwner,
+} from '@mysten/sui.js';
+import { useMemo } from 'react';
 import { Navigate, useNavigate, useParams } from 'react-router-dom';
 
-import TransferNFTForm from './TransferNFTForm';
-import { createValidationSchema } from './validation';
+import { TransferNFTForm } from './TransferNFTForm';
 import Loading from '_components/loading';
-import NFTDisplayCard from '_components/nft-display';
-import { useAppSelector, useAppDispatch, useObjectsState } from '_hooks';
-import { createAccountNftByIdSelector } from '_redux/slices/account';
-import { transferNFT } from '_redux/slices/sui-objects';
-import { DEFAULT_NFT_TRANSFER_GAS_FEE } from '_redux/slices/sui-objects/Coin';
-import PageTitle from '_src/ui/app/shared/PageTitle';
+import { NFTDisplayCard } from '_components/nft-display';
+import Overlay from '_components/overlay';
+import { useAppSelector, useGetObject } from '_hooks';
 
-import type { SerializedError } from '@reduxjs/toolkit';
-import type { FormikHelpers } from 'formik';
-
-import st from './TransferNFTForm.module.scss';
+function NftTransferPage() {
+    const { nftId } = useParams();
+    const address = useAppSelector(({ account: { address } }) => address);
 
-const initialValues = {
-    to: '',
-};
+    // verify that the nft is owned by the user and is transferable
+    const { data: objectData, isLoading } = useGetObject(nftId!);
```

Comment:
same here better default to ''

Reply:
why is it better?

Reply:
since `useGetObject` expects a string it can assume it's safe to handle the input as string. So it could do something like `str.startsWith()` and then it can fail, right?